pkg/config/config.go

```diff
@@ -81,7 +67,7 @@ func (c *Dumper) Dump(configs []v1.Config) error {
 	return nil
 }
 
-func (c *Dumper) WriteFiles() (err error) {
+func (c *Dumper) WriteFiles() error {
 	if c.Configs == nil {
 		logrus.Debug("empty config found")
 		return nil
```

Comment:
1. I am just wondering if `return nil` is reasonable when the `configs==nil`. 
2. Since Dumper has a field of config with slice type
```
type Dumper struct {
	Configs []v1.Config
	Cluster *v2.Cluster
}
```
And at least we should judge by `if len(c.Configs) == 0` rather than `if c.Configs == nil `

Reply:
yes ,only need to judge the input parameter when dumping, not `WriteFiles` func.